flox.nix

```diff
@@ -29,7 +29,7 @@
         (system: (inputs.flox-extras.plugins.catalog {
           catalogDirectory = inputs.catalog + "/render/${system}";
```

Comment:
```suggestion
          catalogDirectory = inputs.catalog + "/catalog/${system}";
```
maybe? should this be updated?

Reply:
I thought we already had.... this was only intended to suggest consistency.

Reply:
It's still `render` (https://github.com/flox/floxpkgs/tree/publish/render)
But i can make a PR to go along